sdks/java/io/hbase/src/test/java/org/apache/beam/sdk/io/hbase/HBaseIOTest.java

```diff
@@ -99,21 +99,21 @@
 
     @BeforeClass
     public static void beforeClass() throws Exception {
-        LOG.info("Starting HBase Embedded Server (HBaseTestUtility)");
+        System.out.println("Starting HBase Embedded Server (HBaseTestUtility)");
         conf.setInt(HConstants.HBASE_CLIENT_RETRIES_NUMBER, 1);
```

Comment:
I think that ultimately what you want is to, in your configuration, set the HBase server name to localhost or 127.0.0.1